WooCommerce/Classes/ViewRelated/ReusableViews/LearnMoreTableViewCell.swift

```diff
@@ -1,12 +1,37 @@
 import UIKit
 
 /// A table view cell with linkable text prompting users to learn more
+///
 class LearnMoreTableViewCell: UITableViewCell {
     @IBOutlet private weak var learnMoreButton: UIButton!
-    @IBOutlet weak var learnMoreTextView: UITextView!
+    @IBOutlet private weak var learnMoreTextView: UITextView!
+
+    private var onUrlPressed: ((_ url: URL) -> Void)?
 
     override func awakeFromNib() {
         super.awakeFromNib()
+        configureCell()
+    }
+
+    private func configureCell() {
         learnMoreButton.setImage(.infoOutlineImage, for: .normal)
+        learnMoreTextView.tintColor = .textLink
+        learnMoreTextView.linkTextAttributes = [
+            .foregroundColor: UIColor.textLink,
+            .underlineColor: UIColor.clear
+        ]
```

Comment:
I think the delegate for `learnMoreTextView` is missing here, WDYT? 🤔

Reply:
I don't think so? `configureCell` is just about setting things up from the storyboard. Setting the text and delegate is supported through the public `configure` method below

Reply:
Oh! 🤦 I see now. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4682/commits/34b050807c24a9aa88bf8df35885ea1e20921860 - I also noticed that with this fix we are getting the sheet style presentation (the un-delegated behavior uses Safari - which I didn't appreciate was due to the un-delegated behavior)